pkgs/development/python-modules/reprint/default.nix

```diff
@@ -0,0 +1,20 @@
+{ lib, fetchurl, buildPythonPackage, python3Packages }:
+
+buildPythonPackage rec {
+  pname = "reprint";
+  version = "0.6.0";
+  src = fetchurl {
+    url =
+      "https://files.pythonhosted.org/packages/ab/30/a742e69e37d4148dfa89acb6aa2a82ea298d1dfea02c6e9b2a43d4ed1065/reprint-0.6.0-py2.py3-none-any.whl";
+    sha256 = "DrxNwhrvyBAgXz/MIXg+qRu5YpKoUAZjSC6GSYm/9U0=";
+  };
```

Comment:
Any reasons for not getting this via `fetchPypi` or from the source repo directly?

Reply:
As previously mentioned there were issues using `fetchPypi` I can try with source repo if that's preferred.

Reply:
I missed those mentions, sorry. But yeah, I think upstream repo is prefered.

Reply:
> I missed those mentions, sorry. But yeah, I think upstream repo is prefered.

@NobbZ, for verilogae there is no longer an upstream. It seems to only be hosted on PyPi. In this case is it okay to keep `fetchurl`?

Reply:
If there is no other way to retrieve it, then there is no other way, at the same time I am not sure if I would hold tight on this overall if a dependencies is not only abondoned by upstream but even got deleted…

Reply:
@NobbZ so I should abandon this PR?